buildSrc/build.gradle.kts

```diff
@@ -1,11 +1,7 @@
 repositories {
-    jcenter()
+    mavenCentral()
 }
 
 plugins {
-    `kotlin-dsl`
+    `embedded-kotlin`
```

Comment:
I like `embedded-kotlin` better because it doesn't apply the SamWithReceiver plugin https://kotlinlang.org/docs/sam-with-receiver-plugin.html, which can be surprising. Both work though so feel free to revert